src/app/db/migrations/versions/20260527_1715_create_ai_usage_log.py

```
@@ -0,0 +1,76 @@
+"""create ai_suggestion_usage_log
+
+Revision ID: 8b1f2e4c9a7d
+Revises: d4a2c8a0705b
+Create Date: 2026-05-27 17:15:00.000000+00:00
+
+"""
+
+from typing import Sequence, Union
```

Ruff (0.15.17)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)